WebContent/WEB-INF/jsp/user/profile.jsp

```diff
@@ -6,6 +6,14 @@
 
 <t:layout>
 	<div class="row">
+		
```

Comment:
Mesajele sunt deja incluse in tag-ul de header.jsp (https://github.com/JavaHaterNLovers/ConferenceManagementSystem/blob/2dc2c69afbf5ab1c6d3ca863167c075da2298e49/WebContent/WEB-INF/tags/header.tag).

Numele variabilei e flashMessage, este un exemplu cum sa folosesti in submit-ul la register.